grafana-toolkit node-sass build error: ‘remove_cv_t’ is not a member of ‘std’;

**What happened**:
"yarn install" build of a fresh plugin created with grafana-toolkit 7.3.7 fails during node-sass build.

**How to reproduce it (as minimally and precisely as possible)**:

Following per instructions at https://grafana.com/tutorials/build-a-panel-plugin/ :

```
npx @grafana/toolkit plugin:create my-plugin
cd my-plugin
yarn install
```
Fails during the build of node-sass with the error shown below regarding unknown C++ template "remove_cv_t":

```
  g++ '-DNODE_GYP_MODULE_NAME=binding' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-D__STDC_FORMAT_MACROS' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DBUILDING_NODE_EXTENSION' -I/home/akira/.node-gyp/16.12.0/include/node -I/home/akira/.node-gyp/16.12.0/src -I/home/akira/.node-gyp/16.12.0/deps/openssl/config -I/home/akira/.node-gyp/16.12.0/deps/openssl/openssl/include -I/home/akira/.node-gyp/16.12.0/deps/uv/include -I/home/akira/.node-gyp/16.12.0/deps/zlib -I/home/akira/.node-gyp/16.12.0/deps/v8/include -I../../nan -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++14 -std=c++0x -MMD -MF ./Release/.deps/Release/obj.target/binding/src/binding.o.d.raw   -c -o Release/obj.target/binding/src/binding.o ../src/binding.cpp
In file included from /home/akira/.node-gyp/16.12.0/include/node/v8.h:30,
                 from /home/akira/.node-gyp/16.12.0/include/node/node.h:63,
                 from ../../nan/nan.h:56,
                 from ../src/binding.cpp:1:
/home/akira/.node-gyp/16.12.0/include/node/v8-internal.h: In function ‘void v8::internal::PerformCastCheck(T*)’:
/home/akira/.node-gyp/16.12.0/include/node/v8-internal.h:492:38: error: ‘remove_cv_t’ is not a member of ‘std’; did you mean ‘remove_cv’?
  492 |             !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
      |                                      ^~~~~~~~~~~
      |                                      remove_cv
/home/akira/.node-gyp/16.12.0/include/node/v8-internal.h:492:38: error: ‘remove_cv_t’ is not a member of ‘std’; did you mean ‘remove_cv’?
  492 |             !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
      |                                      ^~~~~~~~~~~
      |                                      remove_cv
/home/akira/.node-gyp/16.12.0/include/node/v8-internal.h:492:50: error: template argument 2 is invalid
  492 |             !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
      |                                                  ^
/home/akira/.node-gyp/16.12.0/include/node/v8-internal.h:492:63: error: ‘::Perform’ has not been declared
  492 |             !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
      |                                                               ^~~~~~~
...
...
...
...
make: Leaving directory '/var/lib/grafana/plugins/sparkline-stack/node_modules/node-sass/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/var/lib/grafana/plugins/sparkline-stack/node_modules/node-gyp/lib/build.js:262:23)
{
  "author": "Grafana Labs",
  "license": "Apache-2.0",
  "name": "@grafana/toolkit",
  "version": "7.3.7",
  "description": "Grafana Toolkit",
  "keywords": [
    "grafana",
    "cli",
    "plugins",
    "typescript"
  ],
  "repository": {
    "type": "git",
    "url": "http://github.com/grafana/grafana.git",
    "directory": "packages/grafana-toolkit"
  },
  "bin": {
    "grafana-toolkit": "./bin/grafana-toolkit.js"
  },
  "scripts": {
    "build": "grafana-toolkit toolkit:build",
    "clean": "rimraf ./dist ./compiled",
    "precommit": "npm run lint & npm run typecheck",
    "typecheck": "tsc --noEmit"
  },
  "main": "src/index.ts",
  "dependencies": {
    "@babel/core": "7.9.0",
    "@babel/preset-env": "7.9.0",
gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Linux 5.11.0-37-generic
gyp ERR! command "/usr/bin/node" "/var/lib/grafana/plugins/sparkline-stack/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /var/lib/grafana/plugins/sparkline-stack/node_modules/node-sass
gyp ERR! node -v v16.12.0
gyp ERR! node-gyp -v v3.8.0
```

**Anything else we need to know?**:

**Environment**:
- Grafana-toolkit 7.3.7
- OS Grafana is installed on: Ubuntu 21.04
- npm version: 8.1.0
- yarn version: 1.22.17
- Grafana version: 8.2.1

--- a/packages/grafana-toolkit/src/cli/tasks/plugin.create.ts
+++ b/packages/grafana-toolkit/src/cli/tasks/plugin.create.ts
@@ -11,6 +11,7 @@ import {
   promptPluginType,
   removeGitFiles,
   verifyGitExists,
+  removeLockFile,
 } from './plugin/create';
 import { Task, TaskRunner } from './task';
 
@@ -40,10 +41,14 @@ const pluginCreateRunner: TaskRunner<PluginCreateOptions> = async ({ name }) => 
   // 5. Update json files (package.json, src/plugin.json)
   await prepareJsonFiles({ type: type, pluginDetails, pluginPath: destPath });
 
-  // 6. Remove cloned repository .git dir
+  // 6. Starter templates include `yarn.lock` files which will rarely (if ever) be in sync with `latest` dist-tag
+  // so best to remove it after cloning.
+  removeLockFile({ pluginPath: destPath });
+
+  // 7. Remove cloned repository .git dir
   await removeGitFiles(destPath);
 
-  // 7. Promote Grafana Tutorials :)
+  // 8. Promote Grafana Tutorials :)
   printGrafanaTutorialsDetails(type);
 };
 

--- a/packages/grafana-toolkit/src/cli/tasks/plugin/create.ts
+++ b/packages/grafana-toolkit/src/cli/tasks/plugin/create.ts
@@ -1,6 +1,6 @@
 import chalk from 'chalk';
 import commandExists from 'command-exists';
-import { promises as fs, readFileSync } from 'fs';
+import { promises as fs, readFileSync, existsSync, unlinkSync } from 'fs';
 import { prompt } from 'inquirer';
 import { kebabCase } from 'lodash';
 import path from 'path';
@@ -177,6 +177,7 @@ export const printGrafanaTutorialsDetails = (type: PluginType) => {
   console.group();
   console.log();
   console.log(chalk.bold.yellow(`Congrats! You have just created ${PluginNames[type]}.`));
+  console.log('Please run `yarn install` to install frontend dependencies.');
   console.log();
   if (type !== 'backend-datasource-plugin') {
     console.log(`${PluginNames[type]} tutorial: ${TutorialPaths[type]}`);
@@ -188,3 +189,10 @@ export const printGrafanaTutorialsDetails = (type: PluginType) => {
   console.groupEnd();
 };
 /* eslint-enable no-console */
+
+export const removeLockFile = ({ pluginPath }: { pluginPath: string }) => {
+  const lockFilePath = path.resolve(pluginPath, 'yarn.lock');
+  if (existsSync(lockFilePath)) {
+    unlinkSync(lockFilePath);
+  }
+};

